about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About - Capstone Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+    <div class="blend display-none" id="menuBackground"></div>
+  <div class="menu-dropdown display-none" id="menuContent">
+    <button class="closeButton display-none">&#10799;</button>
+    <ul>
+      <li><a class="dropdowns display-none" href="about.html">About</a></li>
+      <li><a class="dropdowns display-none" href="index.html#program">Program</a></li>
+      <li><a class="dropdowns display-none" href="#sponsors">Sponsor</a></li>
+    </ul>
+  </div>
+
+    <header>
+        <nav>
+          <div class="desktop-nav">
+              <a href="index.html"><img class="home-logo" src="images\chess-logo.png" alt="main logo" /></a>
+            <div class="links">
+              <a id="home-link" href="about.html">About</a>
+              <a href="index.html#program">Program</a>
+              <a href="#">Join</a>
+              <a href="#sponsors">Sponsor</a>
+              <a href="#">News</a>
+            </div>
+            <div class="campaign-container">
+              <button class="campaign-button">Chess Summit</button>
+            </div>
+          </div>
+        </nav>
```

Comment:
- [ ] Hey @Boku52500 you did a great job on this project. 👏 However, let's make it look more like the design template. I would recommend adding the `nav bar ` with the social media icons at the top, like in the template.

### The template design 👇️
![Screenshot from 2022-10-21 01-00-48](https://user-images.githubusercontent.com/89228743/197087657-dd5d4ef7-c90d-4a5f-8b54-0ae0f9cf8e3a.png)
### Your design 👇️
![Screenshot from 2022-10-21 01-07-17](https://user-images.githubusercontent.com/89228743/197087793-6ed0ca73-95bc-4fb4-946a-395f8d77c294.png)